index.html

```diff
@@ -227,6 +227,10 @@ <h2 id="x2.link-type-preload">Link type "<code>preload</code>"</h2>
         memory cache and be re-usable at least once within the lifetime of a
         fetch group.</p>
       </div>
+      <div class="note">
+        Link HTTP header should trigger preloads on responses to navigational
```

Comment:
Technically, I think this could be read as no preload for worker responses? The issue is that "subresource" has specific definition in Fetch. How about.. 

> Link HTTP response header should be processed for all [destinations](https://fetch.spec.whatwg.org/#subresource-request).

WDYT?